lib/src/sources/lparstat_source.cc

```diff
@@ -40,7 +40,7 @@ namespace whereami { namespace sources {
     lparstat_data const* lparstat::data()
     {
         if (!data_) {
-            data_.reset(new lparstat_data);
+            data_.reset(new lparstat_data {"", 0, 0, 0});
```

Comment:
I think it would be better to just give `lparstat_data` a proper constructor, so that it is always correctly initialized